agu_oss/clean_data.py

```diff
@@ -23,7 +23,7 @@ def open_and_clean(url):
                           skiprows=69,
                           delim_whitespace=True)
 
-    all_cols = temp_df.columns[1:]
+    all_cols = temp_df.columns[2:]
```

Comment:
@AllisonCh you  have successfully broken the code with your PR following the activity - nice work :)